apps/designer/package.json

```diff
@@ -82,7 +82,7 @@
     "react-dom": "^17.0.2",
     "react-error-boundary": "^3.1.4",
     "react-hotkeys-hook": "^3.4.7",
-    "react-router-dom": "^6.5.0",
+    "react-router-dom": "^6.7.0",
```

Comment:
"6.7.0" otherwise pnpm won't downgrade caz 6.7.0 matches ^6.8.0 which is same as 6.x.x